.packit.yaml

```diff
@@ -96,7 +95,7 @@ jobs:
     trigger: release
     dist_git_branches:
       - fedora-all
-      - epel-9
+      - epel-10-all
```

Comment:
I would rather do this (same for `propose_downstream`):

```suggestion
      epel-10:
        fast_forward_merge_into:
          - epel-10-branched
```

Reply:
But if we are not doing it for Fedora on purpose, then it makes sense to have divergent branches in EPEL 10 too.

Reply:
Yes, I thought that we could start with the same commits, until we can at least. And than when we are forced we can diverge.

Reply:
I mean, without `fast_forward_merge_into`, the `epel10.0` branch will have different commit hashes than `epel10`.